in_progress/roles.md

```diff
@@ -0,0 +1,89 @@
+# Abstract
+
+The purpose of this document is to formalize the governance process used by the `scverse` project, to clarify how decisions are made and how the various elements of our community interact.
+
+As a whole, we are a consensus-based community project.
+However, any subproject within the scverse can have their own governance system as long as they do not conflict with the rules set out here.
+Anyone with an interest in the project can join the community, contribute to the project design, and participate in the decision making process.
+This document describes how that participation takes place, how to find consensus, and how deadlocks are resolved.
+
+# Roles And Responsibilities
+
+## The Community
+
+The scverse community consists of anyone using or working with the project in any way.
```

Comment:
> Does being part of the community entail being part of the GH organization?

From the meeting today:

We want something to signify contributors to the ecosystem. This might be as org member, otherwise we need a different way to track this.

Let's come up with what the "contributor"/ "developer" role is, then come back to this?

I think members of the organization will be in the "contributor" role, but we will follow

Reply:
agree with contributor v. developer role.